Documentation/articles/getting_started/2_creating_a_new_project_vs.md

```diff
@@ -1,20 +1,20 @@
 # Creating a Project with Visual Studio 2019
 
-This guide will walk you through building a starter game with MonoGame using a Windows and Visual Studio 2019.
+This guide will walk you through building a starter game with MonoGame using Windows and Visual Studio 2019.
 
 Start Visual Studio and select **New Project...** in the upper left corner.
 
 ![New Solution](~/images/getting_started/vswin-mg-new-1.png)
 
-Now you should see a "New Project" dialog pop up, from here select **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**.
+You should see the "Create a new project" dialog pop up. From here, select the **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**.
 
 ![New Template](~/images/getting_started/vswin-mg-new-2.png)
 
-Next type in the name that you wish to give your project, for this tutorial let's just use **Pong** (*note* you should not use spaces when naming the project). After you've entered the name, click on the **...** button next to the location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
+Next, type in a name for your project. For this tutorial, **Pong** will be used (note: project names cannot contain spaces). After you've entered the name, click on the **...** button next to the Location text field and choose a directory to contain your project. Finally, click **OK** to create the project.
 
 ![Project Name](~/images/getting_started/vswin-mg-new-3.png)
 
-If everything went correctly, you should see an **Pong** project open up like in the picture below. To run your game simply press the big **Play Button** in the toolbar or press **F5**.
+If everything went correctly, you should see a project named **Pong** (or whatever name you decided on) open up like in the picture below. To run your game, simply press the big **Play Button** in the toolbar, or press **F5**.
```

Comment:
> (or whatever name you decided on)

Feels redundant as we specified that we will use Pong as our project name for the tutorial.